pkg/utils/cluster.go

```diff
@@ -21,6 +37,17 @@ func IsSingleNodeCluster(c client.Client) (bool, error) {
 		glog.Infof("IsSingleNodeCluster(): one node found in the cluster")
 		return true, nil
 	}
-
 	return false, nil
 }
+
+func openshiftSNOClusterStatus(c client.Client) (bool, error) {
```

Comment:
question are we still need this function?

if the CLUSTER_TYPE is openshift and we have 1 node can we call it an SNO?